pfcpiface/up4.go

```diff
@@ -74,6 +74,18 @@ type meter struct {
 	downlinkCellID uint32
 }
 
+type tunnelPeer struct {
+	id uint8
+	// refCount keeps track of F-SEIDs (UE sessions) using this tunnel peer.
+	// It's implemented as Set to avoid duplicated F-SEIDs.
```

Comment:
Since tunnel peers come from FARs, and since there e can be multiple FARs for the same F-SEID,  should we use F-SEID + FAR ID to keep track of the ref count? Also, given the implementation, the name `refCount` is a bit misleading. How about `usedBy`? If you prefer `refCount`, that's fine.

Reply:
I know that for now we install and remove all FARs at session creation/deletion, but in the near future, we will need to support dedicated bearers, i.e., dynamic FAR creation. The F-SEID + FAR ID solution should make this change future-proof.

Reply:
Good point.